### PR TITLE
Preview Container Support

### PR DESCRIPTION
**Pull request by drucelweisse**
Made several things:

- Added all traits and tag layout
- Fixed iPad transition gesture touches
- Added A11yContainer edit view
- Change some models

Simulator Screen Shot - iPhone 14 Pro - 2023-01-03 at 22 40 53

Simulator Screen Shot - iPhone 14 Pro - 2023-01-03 at 22 41 18

### CODE REVIEW

**Comment by drucelweisse** [comment]
Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2023-01-04 at 15 10 26
Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2023-01-04 at 15 11 02

File: VoiceOver Preview/PreviewFeatures/Sources/DesignPreview/Main/Transition/SideTransitionPresentationController.swift [hunk 1]
```diff
@@ -57,6 +57,7 @@ class SideTransitionPresentationController: UIPresentationController {
             target: self,
             action: #selector(dismiss))
         dismissingView.addGestureRecognizer(dismissingTap)
+        container.bringSubviewToFront(presentedView!)
```
Comment (akaDuality) [inline VoiceOver Preview/PreviewFeatures/Sources/DesignPreview/Main/Transition/SideTransitionPresentationController.swift:60]: We should just call `addDismissingView(to: containerView!)` at `presentationTransitionWillBegin` before we call `containerView!.addSubview(presentedView!)`

**Review by akaDuality — APPROVED** [review]
A lot of work, awesome! I request one little change, can you fix it this PR?

File: VoiceOver Preview/PreviewFeatures/Sources/DesignPreview/Main/Transition/SideTransitionPresentationController.swift (hunk 1 above)
Comment (drucelweisse) [reply VoiceOver Preview/PreviewFeatures/Sources/DesignPreview/Main/Transition/SideTransitionPresentationController.swift]: fixed in https://github.com/VODGroup/VoiceOverDesigner/pull/182/commits/c537f2ef1b220f4fae1334a2541bcb5889ace747

**Review by akaDuality — APPROVED** [review]

**Pull request merged**